scripts/update-dependencies.sh

```
@@ -17,3 +17,8 @@ cd vitepress-docs/
 pnpm up --latest
 pnpm add -D wrangler@latest
 cd ..
+
+cd e2e/
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)